### DISCUSSION TITLE
v1.0.0-beta.6

### DISCUSSION THREAD

**Discussion by EvanMulawski** [EvanMulawski/FanControl.CorsairLink#8] [Announcements]
- Commander CORE: Improves device communication resiliency.
- Commander PRO: Fixes an issue where speed channel power would not be properly set on startup.

<hr /><em>This discussion was created from the release <a href='https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.0.0-beta.6'>v1.0.0-beta.6</a>.</em>

**Comment by dshk0718** [comment]
I would like to confirm here that with this beta 6, I can finally use FanControl with SignalRGB with no issues! Nicely done!
